### ISSUE TITLE
Is this ready for production,

### ISSUE THREAD

**Issue by kchizi** [revant/consolidation#1]

**Comment by riddhimaan** [comment]
No. We did not receive any funding to keep this project going. The reports are copied from the existing ones and a concept doctype has been created.
